.travis.yml

```diff
@@ -9,7 +9,7 @@ branches:
   only:
     - master
 install:
-  - curl -o dotnet-sdk.tar.gz -sSL https://download.visualstudio.microsoft.com/download/pr/efa6dde9-a5ee-4322-b13c-a2a02d3980f0/dad445eba341c1d806bae5c8afb47015/dotnet-sdk-3.0.100-preview-010184-linux-x64.tar.gz
+  - curl -o dotnet-sdk.tar.gz -sSL https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/3.0.1xx/dotnet-sdk-latest-linux-x64.tar.gz
```

Comment:
we should use a fixed version here, not "latest"  to prevent sudden breakages. Or is this just temporary because the actual preview3 archive doesn't exist yet? (if so, please add a TODO).

Reply:
Agreed

Reply:
Using global.json to lock the version of the SDK and added scripts for installation.